travis/script.aarch64-gcc.sh

```diff
@@ -2,6 +2,7 @@
 set -ev
 cd /build/build-cross
 make -j 2 all
+export OMP_WAIT_POLICY=passive
```

Comment:
What is this for?

Reply:
This is for changing OpenMP setting.
With the default setting, it does not run fast if there are many OpenMP processes running simultaneously.